config.toml

```diff
@@ -1,7 +1,13 @@
 # filename: config.toml
 
 [p2p]
-BootstrapNodes=["/ip4/104.131.131.82/tcp/4001/ipfs/QmaCpDMGvV2BGHeYERUEnRQAwe3N8SzbUtfsmvsqQLuvuJ", "/ip4/104.236.179.241/tcp/4001/ipfs/QmSoLPppuBtQSGwKDZT2M73ULpjvfd3aZ6ha4oFGL1KrGM",]
+# Alexander testnet bootstrap nodes
```

Comment:
Rather than just inputting Alexander bootnodes here, we should probably just create a command flag for `--testnet alexander` where the config used includes these bootnodes and whatever other configurations required. Given the `config.toml` is not required to run and the default start will be to connect to mainnet we should have a command option to connect to the testnet. Thoughts?

Reply:
I like this idea! Maybe lets create an issue in the meanwhile so we can properly consider how we might implement different network configs

Reply:
yeah I like that idea, let's work on that